example/advection/advection_package.cpp

```diff
@@ -400,9 +446,9 @@ Real AdvectionHst(MeshData<Real> *md) {
   // weighting needs to be applied in the reduction region.
   const bool volume_weighting = std::is_same<T, Kokkos::Sum<Real, HostExecSpace>>::value;
 
-  pmb->par_reduce(
-      PARTHENON_AUTO_LABEL, 0, advected_pack.GetDim(5) - 1, kb.s, kb.e, jb.s, jb.e, ib.s,
-      ib.e,
+  parthenon::par_reduce(
+      parthenon::loop_pattern_mdrange_tag, PARTHENON_AUTO_LABEL, DevExecSpace(), 0,
+      advected_pack.GetDim(5) - 1, kb.s, kb.e, jb.s, jb.e, ib.s, ib.e,
```

Comment:
This was requested by @Yurlungur earlier because the loop isn't associated with `pmb`. He then also suggested adding default `parthenon::par_*` in the future to avoid having to be more verbose like you say.

I think the switch to terse `parthenon::par_reduce` would ultimately be reasonable, so I'm inclined to leave this change as is in anticipation of that unless you feel strongly.